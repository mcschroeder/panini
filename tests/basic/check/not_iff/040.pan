not : (x:bool) -> {y:bool|y = true <=> x = false}

f40 : {b:bool|false}
f40 = not true

f41 : {b:bool|false}
f41 = not false

