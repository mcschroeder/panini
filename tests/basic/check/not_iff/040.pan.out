tests/basic/check/not_iff/040.pan:4:1-4: error: unable to verify type of f40
tests/basic/check/not_iff/040.pan:7:1-4: error: unable to verify type of f41